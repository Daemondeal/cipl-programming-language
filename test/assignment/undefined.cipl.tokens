Token          Lexeme         Line 
IDENTIFIER     undef          1    
EQUAL          =              1    
STRING         "what"         1     what
LINE_END       \n             1    
EOF                           1    
