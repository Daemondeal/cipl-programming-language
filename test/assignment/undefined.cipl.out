::stdout::

::stderr::
Error: Undefined variable 'undef'. at line 1
